s3backgrounddelete/s3backgrounddelete/cortx_s3_config.py

```diff
@@ -353,6 +353,14 @@ def get_max_keys(self):
         else:
             return 1000
 
+    def get_threshold(self):
+        """Return the threshold for max."""
+        threshold = self.s3confstore.get_config('indexid>threshold')
+        if threshold:
+            return int(threshold)
+        else:
+            return 500
```

Comment:
write a constant in constants.py file and use the constant here rather than hardcoding.

Reply:
It is done in the same way for all the integer values in config file, not sure why do we need to write a constant and use the constant here.

Reply:
done